addons/delivery/models/stock_picking.py

```diff
@@ -83,12 +90,24 @@ def _prepare_shipping_invoice_line(self, picking, invoice):
 
         return res
 
-    @api.one
+    @api.model
+    def _invoice_create_line(self, moves, journal_id, inv_type='out_invoice'):
+        InvoiceLine = self.env['account.invoice.line']
+        invoice_ids = super(StockPicking, self)._invoice_create_line(moves, journal_id, inv_type=inv_type)
+        for move in moves:
+            for invoice in move.picking_id.sale_id.invoice_ids.filtered(lambda invoice: invoice.id in invoice_ids):
+                invoice_line = move.picking_id._prepare_shipping_invoice_line(invoice)
+                if invoice_line:
+                    InvoiceLine.create(invoice_line)
+        return invoice_ids
+
+    @api.multi
```

Comment:
hello @mge-odoo ,
IF I can use @api.one it's return the list of record set and i have to access this with list[0] and @api.multi is just return the record set so it's better to use api.multi and just check for single record using ensure_one(). so we need not to iterate the list .
Thanks